builtin/common/vector.lua

```diff
@@ -138,3 +138,8 @@ function vector.divide(a, b)
 			z = a.z / b}
 	end
 end
+
+function vector.sort(a, b)
+	return {x = math.min(a.x, b.x), y = math.min(a.y, b.y), z = math.min(a.z, b.z)},
+		{x = math.max(a.x, b.x), y = math.max(a.y, b.y), z = math.max(a.z, b.z)}
+end
```

Comment:
"Sort" is good terminology for this and is i believe used in the engine, ok for me.